accessories/heater-cooler.js

```diff
@@ -150,34 +150,33 @@ class HeaterCoolerAccessory extends BroadlinkRMAccessory {
    *
    * Prerequisites: this.state is updated with the latest requested value for Target Heater Cooler State
    * @param {any} hexData The decoded data that is passed in by handler
-   * @param {int} previousValue Previous value for targetHeaterCoolerState  
+   * @param {int} previousValue Previous value for targetHeaterCoolerState
    */
   async setTargetHeaterCoolerState(hexData, previousValue) {
     const { config, data, state } = this
     const { internalConfig } = config
     const { available } = internalConfig
-    const { targetHeaterCoolerState, heatingThresholdTemperature, coolingThresholdTemperature } = state
```

Comment:
@kiwi-cam - I tested a simpler fix for this error, you can change const to let and it worked fine in my tests.

    `let { targetHeaterCoolerState, heatingThresholdTemperature, coolingThresholdTemperature } = state`
    
    I would suggest that as a fix to maintain readability of the file. Let me know if that is fine with you.